packages/next/image.d.ts

```diff
@@ -0,0 +1,2 @@
+export * from './dist/client/image'
```

Comment:
shouldn't this file be automatic?

Reply:
Top level type definitions are not automatically generated by typescript in our case. This is similar to `next/link` etc.